### ISSUE TITLE
Many DFC lack faceManaValue and/or faceConvertedManaCost used by Cockatrice

### ISSUE THREAD

**Issue by CTimmerman** [mtgjson/mtgjson#1121]
#### Version(s) tested against

Whatever is in Cockatrice 2.9.0 (2023-09-14).

#### Description of Bug

Bruna, the Fading Light has `<manacost>5WW</manacost>` but `<cmc></cmc>`.
Gisela, the Broken Blade has `<manacost>2WW</manacost>` but `<cmc></cmc>`.
Likewise 193 other cards, like The Mightstone and Weakstone.

I don't know where to report for upstream fix, but isn't `cmc` a field calculated from `manacost`, anyway? Your service could then detect and/or fix those missing values.

**Comment by ZeldaZach** [comment]
This appears to be a Cockatrice issue, not a MTGJSON issue

**Comment by CTimmerman** [comment]
Indeed, it's https://github.com/Cockatrice/Cockatrice/issues/4271
Thanks for looking. My VS Code can barely handle the 300+ MB JSON.

**Comment by CTimmerman** [comment]
Going by that issue's PR, there does appear to be an issue with MTGJSON: `faceName` is present but `faceManaValue` is not.

**Comment by tooomm** [comment]
It looks like the data in `AllPrintings.json` is fine at first look:

```
"convertedManaCost": 7.0,
"faceConvertedManaCost": 7.0,
"faceManaValue": 7.0,
"faceName": "Realm-Cloaked Giant",
"layout": "adventure",
"manaCost": {5}{W}{W},
"manaValue": 7.0,
"name": "Realm-Cloaked Giant // Cast Off",
```
```
"convertedManaCost": 7.0,
"faceConvertedManaCost": 5.0,
"faceManaValue": 5.0,
"faceName": "Cast Off",
"manaValue": 7.0,
```

<details>
<summary>Full card data from json (5.2.2+20231007)</summary>

```
{
  "artist":"Adam Paquette",
  "artistIds":[
    "89023dad-e6c0-41e0-83fb-eb2bfbbdc3f2"
  ],
  "availability":[
    "paper"
  ],
  "borderColor":"black",
  "colorIdentity":[
    "W"
  ],
  "colors":[
    "W"
  ],
  "convertedManaCost":7.0,
  "edhrecRank":4921,
  "faceConvertedManaCost":7.0,
  "faceManaValue":7.0,
  "faceName":"Realm-Cloaked Giant",
  "finishes":[
    "nonfoil"
  ],
  "foreignData":[
    {
      "faceName":"Realm-Cloaked Giant",
      "language":"German",
      "multiverseId":532668,
      "name":"Realm-Cloaked Giant // Abstreifen",
      "text":"Wachsamkeit",
      "type":"Kreatur — Riese"
    },
    {
      "faceName":"Realm-Cloaked Giant",
      "language":"Spanish",
      "multiverseId":532879,
      "name":"Realm-Cloaked Giant // Quitar los hilos",
      "text":"Vigilancia.",
      "type":"Criatura — Gigante"
    },
    {
      "faceName":"Géant drapé de royaume",
      "language":"French",
      "multiverseId":533090,
      "name":"Géant drapé de royaume // Dévêtir",
      "text":"Vigilance",
      "type":"Créature : géant"
    },
    {
      "faceName":"Realm-Cloaked Giant",
      "language":"Italian",
      "multiverseId":533301,
      "name":"Realm-Cloaked Giant // Gettare Via",
      "text":"Cautela",
      "type":"Creatura — Gigante"
    },
    {
      "faceName":"Realm-Cloaked Giant",
      "language":"Japanese",
      "multiverseId":533512,
      "name":"Realm-Cloaked Giant // 脱ぎ捨て",
      "text":"警戒",
      "type":"クリーチャー — 巨人"
    },
    {
      "faceName":"Realm-Cloaked Giant",
      "language":"Portuguese (Brazil)",
      "multiverseId":533723,
      "name":"Realm-Cloaked Giant // Despojar-se",
      "text":"Vigilância",
      "type":"Criatura — Gigante"
    },
    {
      "faceName":"Realm-Cloaked Giant",
      "language":"Russian",
      "multiverseId":533934,
      "name":"Realm-Cloaked Giant // Отбросить",
      "text":"Бдительность",
      "type":"Существо — Гигант"
    },
    {
      "faceName":"Realm-Cloaked Giant",
      "language":"Chinese Simplified",
      "multiverseId":534145,
      "name":"Realm-Cloaked Giant // 褪下披风",
      "text":"警戒",
      "type":"生物 ～巨人"
    },
    {
      "faceName":"Realm-Cloaked Giant",
      "language":"Chinese Traditional",
      "multiverseId":534356,
      "name":"Realm-Cloaked Giant // 褪下披風",
      "text":"警戒",
      "type":"生物 ～巨人"
    }
  ],
  "frameVersion":"2015",
  "hasFoil":false,
  "hasNonFoil":true,
  "identifiers":{
    "cardKingdomId":"249186",
    "mcmId":"572243",
    "mtgjsonV4Id":"c8ef769b-58aa-5975-84e5-955dcbe6dd27",
    "multiverseId":"532457",
    "scryfallId":"d6fc6ed9-740a-4726-8f2c-5d662636f687",
    "scryfallIllustrationId":"6db841f6-7c2b-4248-8e99-3eb08f381b1c",
    "scryfallOracleId":"a1d17244-9c92-4094-9843-d7ee31c85ea4",
    "tcgplayerProductId":"243817"
  },
  "isReprint":true,
  "isStarter":true,
  "keywords":[
    "Vigilance"
  ],
  "language":"English",
  "layout":"adventure",
  "legalities":{
    "commander":"Legal",
    "duel":"Legal",
    "explorer":"Legal",
    "gladiator":"Legal",
    "historic":"Legal",
    "historicbrawl":"Legal",
    "legacy":"Legal",
    "modern":"Legal",
    "oathbreaker":"Legal",
    "penny":"Legal",
    "pioneer":"Legal",
    "vintage":"Legal"
  },
  "manaCost":"{5}{W}{W}",
  "manaValue":7.0,
  "name":"Realm-Cloaked Giant // Cast Off",
  "number":"70",
  "originalText":"Vigilance",
  "originalType":"Creature — Giant",
  "otherFaceIds":[
    "131f4174-02e8-5435-ad81-e90fc1b849cb"
  ],
  "power":"7",
  "printings":[
    "AFC",
    "ELD",
    "PELD",
    "PRM",
    "WOC"
  ],
  "purchaseUrls":{
    "cardKingdom":"https://mtgjson.com/links/4ae798a8a9ac3a23",
    "tcgplayer":"https://mtgjson.com/links/25b133eb942fc459"
  },
  "rarity":"mythic",
  "rulings":[
    {
      "date":"2019-10-04",
      "text":"An adventurer card is a creature card in every zone except the stack, as well as while on the stack if not cast as an Adventure. Ignore its alternative characteristics in those cases. For example, while it's in your graveyard, Giant Killer is a white creature card whose mana value is 1. It can't be the target of the triggered ability of Mystic Sanctuary."
    },
    {
      "date":"2019-10-04",
      "text":"Casting a card as an Adventure isn't casting it for an alternative cost. Effects that allow you to cast a spell for an alternative cost or without paying its mana cost may allow you to apply those to the Adventure."
    },
    {
      "date":"2019-10-04",
      "text":"If a spell is cast as an Adventure, its controller exiles it instead of putting it into its owner's graveyard as it resolves. For as long as it remains exiled, that player may cast it as a creature spell. If an Adventure spell leaves the stack in any way other than resolving (most likely by being countered or by failing to resolve because its targets have all become illegal), that card won't be exiled and the spell's controller won't be able to cast it as a creature later."
    },
    {
      "date":"2019-10-04",
      "text":"If an adventurer card ends up in exile for any other reason than by exiling itself while resolving, it won't give you permission to cast it as a creature spell."
    },
    {
      "date":"2019-10-04",
      "text":"If an effect copies an Adventure spell, that copy is exiled as it resolves. It ceases to exist as a state-based action; it's not possible to cast the copy as a creature."
    },
    {
      "date":"2019-10-04",
      "text":"If an effect instructs you to choose a card name, you may choose the alternative Adventure name. Consider only the alternative characteristics to determine whether that is an appropriate name to choose."
    },
    {
      "date":"2019-10-04",
      "text":"If an object becomes a copy of an object that has an Adventure, the copy also has an Adventure. If it changes zones, it will either cease to exist (if it's a token) or cease to be a copy (if it's a nontoken permanent), and so you won't be able to cast it as an Adventure."
    },
    {
      "date":"2019-10-04",
      "text":"If you cast an adventurer card as an Adventure, use only its alternative characteristics to determine whether it's legal to cast that spell. For example, if Giant Killer is exiled with the last ability of Vivien, Champion of the Wilds, you can't cast it as Chop Down."
    },
    {
      "date":"2019-10-04",
      "text":"When casting a spell as an Adventure, use the alternative characteristics and ignore all of the card's normal characteristics. The spell's color, mana cost, mana value, and so on are determined by only those alternative characteristics. If the spell leaves the stack, it immediately resumes using its normal characteristics."
    },
    {
      "date":"2019-10-04",
      "text":"You must still follow any timing restrictions and permissions for the creature spell you cast from exile. Normally, you'll be able to cast it only during your main phase while the stack is empty."
    }
  ],
  "securityStamp":"oval",
  "setCode":"AFC",
  "side":"a",
  "sourceProducts":{
    "nonfoil":[
      "c90c5b01-4fd3-5269-8a88-81b80b760b29"
    ]
  },
  "subtypes":[
    "Giant"
  ],
  "supertypes":[
    
  ],
  "text":"Vigilance",
  "toughness":"7",
  "type":"Creature — Giant",
  "types":[
    "Creature"
  ],
  "uuid":"a7c38e21-6688-54a6-ae77-ced41669abe9"
},
{
  "artist":"Adam Paquette",
  "artistIds":[
    "89023dad-e6c0-41e0-83fb-eb2bfbbdc3f2"
  ],
  "availability":[
    "paper"
  ],
  "borderColor":"black",
  "colorIdentity":[
    "W"
  ],
  "colors":[
    "W"
  ],
  "convertedManaCost":7.0,
  "edhrecRank":4921,
  "faceConvertedManaCost":5.0,
  "faceManaValue":5.0,
  "faceName":"Cast Off",
  "finishes":[
    "nonfoil"
  ],
  "foreignData":[
    {
      "faceName":"Abstreifen",
      "language":"German",
      "multiverseId":532668,
      "name":"Realm-Cloaked Giant // Abstreifen",
      "text":"Zerstöre alle Nicht-Riese-Kreaturen. (Schicke diese Karte dann ins Exil. Du kannst sie später als Kreatur aus dem Exil wirken.)",
      "type":"Hexerei — Abenteuer"
    },
    {
      "faceName":"Quitar los hilos",
      "language":"Spanish",
      "multiverseId":532879,
      "name":"Realm-Cloaked Giant // Quitar los hilos",
      "text":"Destruye todas las criaturas que no sean Gigante. (Luego, exilia esta carta. Puedes lanzar la criatura más adelante desde el exilio.)",
      "type":"Conjuro — Aventura"
    },
    {
      "faceName":"Dévêtir",
      "language":"French",
      "multiverseId":533090,
      "name":"Géant drapé de royaume // Dévêtir",
      "text":"Détruisez toutes les créatures non-Géant. (Puis exilez cette carte. Vous pouvez lancer la créature plus tard depuis l'exil.)",
      "type":"Rituel : aventure"
    },
    {
      "faceName":"Gettare Via",
      "language":"Italian",
      "multiverseId":533301,
      "name":"Realm-Cloaked Giant // Gettare Via",
      "text":"Distruggi tutte le creature non Gigante. (Poi esilia questa carta. Puoi lanciare la creatura in seguito dall'esilio.)",
      "type":"Stregoneria — Avventura"
    },
    {
      "faceName":"脱ぎ捨て",
      "language":"Japanese",
      "multiverseId":533512,
      "name":"Realm-Cloaked Giant // 脱ぎ捨て",
      "text":"巨人でないすべてのクリーチャーを破壊する。（その後、このカードを追放する。あなたは後で追放領域にあるこのクリーチャーを唱えてもよい。）",
      "type":"ソーサリー ― 出来事"
    },
    {
      "faceName":"Despojar-se",
      "language":"Portuguese (Brazil)",
      "multiverseId":533723,
      "name":"Realm-Cloaked Giant // Despojar-se",
      "text":"Destrua todas as criaturas não Gigantes. (Depois, exile este card. Você pode conjurar a criatura posteriormente do exílio.)",
      "type":"Feitiço — Aventura"
    },
    {
      "faceName":"Отбросить",
      "language":"Russian",
      "multiverseId":533934,
      "name":"Realm-Cloaked Giant // Отбросить",
      "text":"Уничтожьте все не являющиеся Гигантами существа. (Затем изгоните эту карту. Позже вы можете разыграть эту карту как существо из изгнания.)",
      "type":"Волшебство — Приключение"
    },
    {
      "faceName":"褪下披风",
      "language":"Chinese Simplified",
      "multiverseId":534145,
      "name":"Realm-Cloaked Giant // 褪下披风",
      "text":"消灭所有非巨人生物。（然后放逐此牌。之后你还可以从放逐区中施放该生物。）",
      "type":"法术～历险"
    },
    {
      "faceName":"褪下披風",
      "language":"Chinese Traditional",
      "multiverseId":534356,
      "name":"Realm-Cloaked Giant // 褪下披風",
      "text":"消滅所有非巨人生物。（然後放逐此牌。之後你還可以從放逐區中施放該生物。）",
      "type":"巫術～歷險"
    }
  ],
  "frameVersion":"2015",
  "hasFoil":false,
  "hasNonFoil":true,
  "identifiers":{
    "mcmId":"572243",
    "mtgjsonV4Id":"56a3dd73-6dc7-5062-9d37-e8c55ececc1a",
    "multiverseId":"532457",
    "scryfallId":"d6fc6ed9-740a-4726-8f2c-5d662636f687",
    "scryfallIllustrationId":"6db841f6-7c2b-4248-8e99-3eb08f381b1c",
    "scryfallOracleId":"a1d17244-9c92-4094-9843-d7ee31c85ea4",
    "tcgplayerProductId":"243817"
  },
  "isReprint":true,
  "isStarter":true,
  "language":"English",
  "layout":"adventure",
  "legalities":{
    "commander":"Legal",
    "duel":"Legal",
    "explorer":"Legal",
    "gladiator":"Legal",
    "historic":"Legal",
    "historicbrawl":"Legal",
    "legacy":"Legal",
    "modern":"Legal",
    "oathbreaker":"Legal",
    "penny":"Legal",
    "pioneer":"Legal",
    "vintage":"Legal"
  },
  "manaCost":"{3}{W}{W}",
  "manaValue":7.0,
  "name":"Realm-Cloaked Giant // Cast Off",
  "number":"70",
  "originalText":"Destroy all non-Giant creatures. (Then exile this card. You may cast the creature later from exile.)",
  "originalType":"Sorcery — Adventure",
  "otherFaceIds":[
    "a7c38e21-6688-54a6-ae77-ced41669abe9"
  ],
  "printings":[
    "AFC",
    "ELD",
    "PELD",
    "PRM",
    "WOC"
  ],
  "purchaseUrls":{
    "tcgplayer":"https://mtgjson.com/links/1b2a70e2e2cde795"
  },
  "rarity":"mythic",
  "rulings":[
    {
      "date":"2019-10-04",
      "text":"An adventurer card is a creature card in every zone except the stack, as well as while on the stack if not cast as an Adventure. Ignore its alternative characteristics in those cases. For example, while it's in your graveyard, Giant Killer is a white creature card whose mana value is 1. It can't be the target of the triggered ability of Mystic Sanctuary."
    },
    {
      "date":"2019-10-04",
      "text":"Casting a card as an Adventure isn't casting it for an alternative cost. Effects that allow you to cast a spell for an alternative cost or without paying its mana cost may allow you to apply those to the Adventure."
    },
    {
      "date":"2019-10-04",
      "text":"If a spell is cast as an Adventure, its controller exiles it instead of putting it into its owner's graveyard as it resolves. For as long as it remains exiled, that player may cast it as a creature spell. If an Adventure spell leaves the stack in any way other than resolving (most likely by being countered or by failing to resolve because its targets have all become illegal), that card won't be exiled and the spell's controller won't be able to cast it as a creature later."
    },
    {
      "date":"2019-10-04",
      "text":"If an adventurer card ends up in exile for any other reason than by exiling itself while resolving, it won't give you permission to cast it as a creature spell."
    },
    {
      "date":"2019-10-04",
      "text":"If an effect copies an Adventure spell, that copy is exiled as it resolves. It ceases to exist as a state-based action; it's not possible to cast the copy as a creature."
    },
    {
      "date":"2019-10-04",
      "text":"If an effect instructs you to choose a card name, you may choose the alternative Adventure name. Consider only the alternative characteristics to determine whether that is an appropriate name to choose."
    },
    {
      "date":"2019-10-04",
      "text":"If an object becomes a copy of an object that has an Adventure, the copy also has an Adventure. If it changes zones, it will either cease to exist (if it's a token) or cease to be a copy (if it's a nontoken permanent), and so you won't be able to cast it as an Adventure."
    },
    {
      "date":"2019-10-04",
      "text":"If you cast an adventurer card as an Adventure, use only its alternative characteristics to determine whether it's legal to cast that spell. For example, if Giant Killer is exiled with the last ability of Vivien, Champion of the Wilds, you can't cast it as Chop Down."
    },
    {
      "date":"2019-10-04",
      "text":"When casting a spell as an Adventure, use the alternative characteristics and ignore all of the card's normal characteristics. The spell's color, mana cost, mana value, and so on are determined by only those alternative characteristics. If the spell leaves the stack, it immediately resumes using its normal characteristics."
    },
    {
      "date":"2019-10-04",
      "text":"You must still follow any timing restrictions and permissions for the creature spell you cast from exile. Normally, you'll be able to cast it only during your main phase while the stack is empty."
    }
  ],
  "securityStamp":"oval",
  "setCode":"AFC",
  "side":"b",
  "subtypes":[
    "Adventure"
  ],
  "supertypes":[
    
  ],
  "text":"Destroy all non-Giant creatures. (Then exile this card. You may cast the creature later from exile.)",
  "type":"Sorcery — Adventure",
  "types":[
    "Sorcery"
  ],
  "uuid":"131f4174-02e8-5435-ad81-e90fc1b849cb"
}
```
</details>

<br>

---

<br>

```
"convertedManaCost": 9.0,
"faceConvertedManaCost": 4.0,
"faceManaValue": 4.0,
"faceName": "Dusk",
"layout": "aftermath",
"manaCost": {2}{W}{W},
"manaValue": 9.0,
"name": "Dusk // Dawn",
```
```
"convertedManaCost": 9.0,
"faceConvertedManaCost": 5.0,
"faceManaValue": 5.0,
"faceName": "Dawn",
```

**Comment by CTimmerman** [comment]
@tooomm There is still only one instance of `manaValue` (no `faceManaValue`) in the aforementioned cases, eg 12 of Blex:
```
Found! "name": "Blex, Vexing Pest // Blex, Vexing Pest",
Card 1 {"artist": "Ekaterina Burmak", "artistIds": ["2a9f3780-fcc8-4ec5-b403-ee1d1370d3b4"], "availability": ["paper"], "borderColor": "borderless", "colorIdentity": [], "colors": [], "faceName": "Blex, Vexing Pest", "finishes": ["nonfoil", "foil", "signed"], "frameVersion": "2015", "hasFoil": true, "hasNonFoil": true, "identifiers": {"mtgjsonV4Id": "63f1fbea-07c7-55b3-bcca-d0005757f972", "scryfallId": "9b419cf8-fd48-40fc-95fa-5e1e28ac32f7", "scryfallIllustrationId": "194dcecb-46d4-49a5-bc84-d5a192330972", "scryfallOracleId": "e4ac23cc-b257-4cf9-90df-c40b30cf93c2"}, "language": "English", "layout": "art_series", "name": "Blex, Vexing Pest // Blex, Vexing Pest", "number": "22", "orientation": "horizontal", "otherFaceIds": ["269194ff-4d6c-569a-95c6-e3d710e7ed31"], "reverseRelated": [], "setCode": "ASTX", "side": "a", "signature": "Ekaterina Burmak", "subtypes": [], "supertypes": [], "type": "Card", "types": ["Card"], "uuid": "f494eecd-a50e-56aa-8e23-5685743b03f0"} /CARD
Found! "name": "Blex, Vexing Pest // Blex, Vexing Pest",
Card 2 {"artist": "Ekaterina Burmak", "artistIds": ["2a9f3780-fcc8-4ec5-b403-ee1d1370d3b4"], "availability": ["paper"], "borderColor": "borderless", "colorIdentity": [], "colors": [], "faceName": "Blex, Vexing Pest", "finishes": ["nonfoil", "foil", "signed"], "frameVersion": "2015", "hasFoil": true, "hasNonFoil": true, "identifiers": {"mtgjsonV4Id": "c0e69a22-09dd-5769-8c13-a83a339b40be", "scryfallId": "9b419cf8-fd48-40fc-95fa-5e1e28ac32f7", "scryfallIllustrationId": "c0e428ae-2b15-4605-83a6-851916c62351", "scryfallOracleId": "e4ac23cc-b257-4cf9-90df-c40b30cf93c2"}, "language": "English", "layout": "art_series", "name": "Blex, Vexing Pest // Blex, Vexing Pest", "number": "22", "orientation": "horizontal", "otherFaceIds": ["f494eecd-a50e-56aa-8e23-5685743b03f0"], "reverseRelated": [], "setCode": "ASTX", "side": "b", "signature": "Ekaterina Burmak", "subtypes": [], "supertypes": [], "type": "Card", "types": ["Card"], "uuid": "269194ff-4d6c-569a-95c6-e3d710e7ed31"} /CARD
Found! "name": "Blex, Vexing Pest // Blex, Vexing Pest",
Card 3 {"artist": "Ekaterina Burmak", "artistIds": ["2a9f3780-fcc8-4ec5-b403-ee1d1370d3b4"], "availability": ["paper"], "borderColor": "borderless", "colorIdentity": [], "colors": [], "faceName": "Blex, Vexing Pest", "finishes": ["nonfoil", "foil", "signed"], "frameVersion": "2015", "hasFoil": true, "hasNonFoil": true, "identifiers": {"mtgjsonV4Id": "9599f2d2-3512-584f-b4b4-5b511452fdf0", "scryfallId": "2f6838c7-f934-4237-bdb0-2066e4230ba0", "scryfallIllustrationId": "194dcecb-46d4-49a5-bc84-d5a192330972", "scryfallOracleId": "e4ac23cc-b257-4cf9-90df-c40b30cf93c2"}, "language": "English", "layout": "art_series", "name": "Blex, Vexing Pest // Blex, Vexing Pest", "number": "22s", "orientation": "horizontalstamped", "otherFaceIds": ["0cf1ffcb-20e9-5d9f-9623-4e146855a7fc"], "promoTypes": ["stamped"], "reverseRelated": [], "setCode": "ASTX", "side": "a", "signature": "Ekaterina Burmak", "subtypes": [], "supertypes": [], "type": "Card", "types": ["Card"], "uuid": "39316a2f-a5ae-5b1c-99f3-7d2a03b5a4b0"} /CARD
Found! "name": "Blex, Vexing Pest // Blex, Vexing Pest",
Card 4 {"artist": "Ekaterina Burmak", "artistIds": ["2a9f3780-fcc8-4ec5-b403-ee1d1370d3b4"], "availability": ["paper"], "borderColor": "borderless", "colorIdentity": [], "colors": [], "faceName": "Blex, Vexing Pest", "finishes": ["nonfoil", "foil", "signed"], "frameVersion": "2015", "hasFoil": true, "hasNonFoil": true, "identifiers": {"mtgjsonV4Id": "7ae6c2b5-26c5-5a14-bf4a-89b9fa9f459c", "scryfallId": "2f6838c7-f934-4237-bdb0-2066e4230ba0", "scryfallOracleId": "e4ac23cc-b257-4cf9-90df-c40b30cf93c2"}, "language": "English", "layout": "art_series", "name": "Blex, Vexing Pest // Blex, Vexing Pest", "number": "22s", "orientation": "horizontalstamped", "otherFaceIds": ["39316a2f-a5ae-5b1c-99f3-7d2a03b5a4b0"], "promoTypes": ["stamped"], "reverseRelated": [], "setCode": "ASTX", "side": "b", "signature": "Ekaterina Burmak", "subtypes": [], "supertypes": [], "type": "Card", "types": ["Card"], "uuid": "0cf1ffcb-20e9-5d9f-9623-4e146855a7fc"} /CARD
Found! "name": "Blex, Vexing Pest // Search for Blex",
Card 5 {"artist": "Ekaterina Burmak", "artistIds": ["2a9f3780-fcc8-4ec5-b403-ee1d1370d3b4"], "availability": ["mtgo"], "borderColor": "black", "colorIdentity": ["B", "G"], "colors": ["G"], "convertedManaCost": 3.0, "edhrecRank": 5557, "faceName": "Blex, Vexing Pest", "finishes": ["nonfoil", "foil"], "foreignData": [], "frameVersion": "2015", "hasFoil": true, "hasNonFoil": true, "identifiers": {"mtgjsonV4Id": "a0be2a6a-e798-53a9-b861-56330cf833e9", "mtgoId": "90300", "scryfallId": "5c3fe536-4ec3-4b14-b0ce-c44059d8d0e6", "scryfallIllustrationId": "59e38d12-40fd-46ec-94ca-6ac4b09df2fc", "scryfallOracleId": "b2c53124-e422-4786-8135-67c350d71a36"}, "isOnlineOnly": true, "isPromo": true, "isReprint": true, "language": "English", "layout": "modal_dfc", "leadershipSkills": {"brawl": false, "commander": true, "oathbreaker": false}, "legalities": {"commander": "Legal", "duel": "Legal", "explorer": "Legal", "gladiator": "Legal", "historic": "Legal", "historicbrawl": "Legal", "legacy": "Legal", "modern": "Legal", "oathbreaker": "Legal", "pioneer": "Legal", "vintage": "Legal"}, "manaCost": "{2}{G}", "manaValue": 3.0, "name": "Blex, Vexing Pest // Search for Blex", "number": "90300", "originalReleaseDate": "2021-05-01", "otherFaceIds": ["8687e5e3-82f2-5bd1-9513-db2f7c86633a"], "power": "3", "printings": ["PRM", "PSTX", "STX"], "purchaseUrls": {}, "rarity": "mythic", "rulings": [{"date": "2021-04-16", "text": "If a creature is more than one of the creature types Blex cares about, it may haunt your dreams, but it will get the +1/+1 bonus only once."}, {"date": "2021-04-16", "text": "While resolving Search for Blex, you may put any number of the cards into your hand, even if you don’t have enough life to cover it. We’ll assume you have a plan that’s better than “and then I’ll lose the game.”"}], "securityStamp": "oval", "setCode": "PRM", "side": "a", "subtypes": ["Pest"], "supertypes": ["Legendary"], "text": "Other Pests, Bats, Insects, Snakes, and Spiders you control get +1/+1.\nWhen Blex, Vexing Pest dies, you gain 4 life.", "toughness": "2", "type": "Legendary Creature — Pest", "types": ["Creature"], "uuid": "126ddcec-7bbd-52c6-a88f-1c7276cd5beb"} /CARD
Found! "name": "Blex, Vexing Pest // Search for Blex",
Card 6 {"artist": "Ekaterina Burmak", "artistIds": ["2a9f3780-fcc8-4ec5-b403-ee1d1370d3b4"], "availability": ["mtgo"], "borderColor": "black", "colorIdentity": ["B", "G"], "colors": ["B"], "convertedManaCost": 4.0, "edhrecRank": 5557, "faceName": "Search for Blex", "finishes": ["nonfoil", "foil"], "foreignData": [], "frameVersion": "2015", "hasFoil": true, "hasNonFoil": true, "identifiers": {"mtgjsonV4Id": "1e655dc9-c36d-5a4c-81c0-3616bd38ca69", "mtgoId": "90300", "scryfallId": "5c3fe536-4ec3-4b14-b0ce-c44059d8d0e6", "scryfallIllustrationId": "a9acd45f-85da-4dad-8e85-cd0be900ce3c", "scryfallOracleId": "b2c53124-e422-4786-8135-67c350d71a36"}, "isOnlineOnly": true, "isPromo": true, "isReprint": true, "language": "English", "layout": "modal_dfc", "legalities": {"commander": "Legal", "duel": "Legal", "explorer": "Legal", "gladiator": "Legal", "historic": "Legal", "historicbrawl": "Legal", "legacy": "Legal", "modern": "Legal", "oathbreaker": "Legal", "pioneer": "Legal", "vintage": "Legal"}, "manaCost": "{2}{B}{B}", "manaValue": 4.0, "name": "Blex, Vexing Pest // Search for Blex", "number": "90300", "originalReleaseDate": "2021-05-01", "otherFaceIds": ["126ddcec-7bbd-52c6-a88f-1c7276cd5beb"], "printings": ["PRM", "PSTX", "STX"], "purchaseUrls": {}, "rarity": "mythic", "rulings": [{"date": "2021-04-16", "text": "If a creature is more than one of the creature types Blex cares about, it may haunt your dreams, but it will get the +1/+1 bonus only once."}, {"date": "2021-04-16", "text": "While resolving Search for Blex, you may put any number of the cards into your hand, even if you don’t have enough life to cover it. We’ll assume you have a plan that’s better than “and then I’ll lose the game.”"}], "securityStamp": "oval", "setCode": "PRM", "side": "b", "subtypes": [], "supertypes": [], "text": "Look at the top five cards of your library. You may put any number of them into your hand and the rest into your graveyard. You lose 3 life for each card you put into your hand this way.", "type": "Sorcery", "types": ["Sorcery"], "uuid": "8687e5e3-82f2-5bd1-9513-db2f7c86633a"} /CARD
Found! "name": "Blex, Vexing Pest // Search for Blex",
Card 7 {"artist": "Ekaterina Burmak", "artistIds": ["2a9f3780-fcc8-4ec5-b403-ee1d1370d3b4"], "availability": ["paper"], "borderColor": "black", "colorIdentity": ["B", "G"], "colors": ["G"], "convertedManaCost": 3.0, "edhrecRank": 5557, "faceName": "Blex, Vexing Pest", "finishes": ["foil"], "flavorText": "Pests are typically harvested before they outgrow their enclosures. Blex is far from typical.", "foreignData": [], "frameEffects": ["legendary"], "frameVersion": "2015", "hasFoil": true, "hasNonFoil": false, "identifiers": {"cardKingdomFoilId": "245688", "cardsphereId": "76936", "mcmId": "560565", "mtgjsonV4Id": "b10bbecc-da8e-5513-8e23-6419b9c71c78", "scryfallId": "44fad71f-5928-4ba3-bd53-2dedd24f0277", "scryfallIllustrationId": "59e38d12-40fd-46ec-94ca-6ac4b09df2fc", "scryfallOracleId": "b2c53124-e422-4786-8135-67c350d71a36", "tcgplayerProductId": "237137"}, "isPromo": true, "isReprint": true, "isStarter": true, "language": "English", "layout": "modal_dfc", "leadershipSkills": {"brawl": false, "commander": true, "oathbreaker": false}, "legalities": {"commander": "Legal", "duel": "Legal", "explorer": "Legal", "gladiator": "Legal", "historic": "Legal", "historicbrawl": "Legal", "legacy": "Legal", "modern": "Legal", "oathbreaker": "Legal", "pioneer": "Legal", "vintage": "Legal"}, "manaCost": "{2}{G}", "manaValue": 3.0, "name": "Blex, Vexing Pest // Search for Blex", "number": "148s", "otherFaceIds": ["a187001d-6b68-578c-a66e-656143975e67"], "power": "3", "printings": ["PRM", "PSTX", "STX"], "promoTypes": ["prerelease", "datestamped"], "purchaseUrls": {"cardKingdomFoil": "https://mtgjson.com/links/491edc1e37bc2363", "tcgplayer": "https://mtgjson.com/links/d5d96ec0ad223a62"}, "rarity": "mythic", "rulings": [{"date": "2021-04-16", "text": "If a creature is more than one of the creature types Blex cares about, it may haunt your dreams, but it will get the +1/+1 bonus only once."}, {"date": "2021-04-16", "text": "While resolving Search for Blex, you may put any number of the cards into your hand, even if you don’t have enough life to cover it. We’ll assume you have a plan that’s better than “and then I’ll lose the game.”"}], "securityStamp": "oval", "setCode": "PSTX", "side": "a", "subtypes": ["Pest"], "supertypes": ["Legendary"], "text": "Other Pests, Bats, Insects, Snakes, and Spiders you control get +1/+1.\nWhen Blex, Vexing Pest dies, you gain 4 life.", "toughness": "2", "type": "Legendary Creature — Pest", "types": ["Creature"], "uuid": "0720bc71-56e4-57b2-89c7-01237cf6066a", "watermark": "witherbloom"} /CARD
Found! "name": "Blex, Vexing Pest // Search for Blex",
Card 8 {"artist": "Ekaterina Burmak", "artistIds": ["2a9f3780-fcc8-4ec5-b403-ee1d1370d3b4"], "availability": ["paper"], "borderColor": "black", "colorIdentity": ["B", "G"], "colors": ["B"], "convertedManaCost": 4.0, "edhrecRank": 5557, "faceName": "Search for Blex", "finishes": ["foil"], "foreignData": [], "frameEffects": ["legendary"], "frameVersion": "2015", "hasFoil": true, "hasNonFoil": false, "identifiers": {"cardsphereId": "76936", "mcmId": "560565", "mtgjsonV4Id": "785ec4cf-5799-5eab-b38a-c3be55b5374a", "scryfallId": "44fad71f-5928-4ba3-bd53-2dedd24f0277", "scryfallIllustrationId": "a9acd45f-85da-4dad-8e85-cd0be900ce3c", "scryfallOracleId": "b2c53124-e422-4786-8135-67c350d71a36", "tcgplayerProductId": "237137"}, "isPromo": true, "isReprint": true, "isStarter": true, "language": "English", "layout": "modal_dfc", "legalities": {"commander": "Legal", "duel": "Legal", "explorer": "Legal", "gladiator": "Legal", "historic": "Legal", "historicbrawl": "Legal", "legacy": "Legal", "modern": "Legal", "oathbreaker": "Legal", "pioneer": "Legal", "vintage": "Legal"}, "manaCost": "{2}{B}{B}", "manaValue": 4.0, "name": "Blex, Vexing Pest // Search for Blex", "number": "148s", "otherFaceIds": ["0720bc71-56e4-57b2-89c7-01237cf6066a"], "printings": ["PRM", "PSTX", "STX"], "promoTypes": ["prerelease", "datestamped"], "purchaseUrls": {"tcgplayer": "https://mtgjson.com/links/03304792ddfcd916"}, "rarity": "mythic", "rulings": [{"date": "2021-04-16", "text": "If a creature is more than one of the creature types Blex cares about, it may haunt your dreams, but it will get the +1/+1 bonus only once."}, {"date": "2021-04-16", "text": "While resolving Search for Blex, you may put any number of the cards into your hand, even if you don’t have enough life to cover it. We’ll assume you have a plan that’s better than “and then I’ll lose the game.”"}], "securityStamp": "oval", "setCode": "PSTX", "side": "b", "subtypes": [], "supertypes": [], "text": "Look at the top five cards of your library. You may put any number of them into your hand and the rest into your graveyard. You lose 3 life for each card you put into your hand this way.", "type": "Sorcery", "types": ["Sorcery"], "uuid": "a187001d-6b68-578c-a66e-656143975e67", "watermark": "witherbloom"} /CARD
Found! "name": "Blex, Vexing Pest // Search for Blex",
Card 9 {"artist": "Ekaterina Burmak", "artistIds": ["2a9f3780-fcc8-4ec5-b403-ee1d1370d3b4"], "availability": ["arena", "mtgo", "paper"], "boosterTypes": ["default"], "borderColor": "black", "colorIdentity": ["B", "G"], "colors": ["G"], "convertedManaCost": 3.0, "edhrecRank": 5557, "faceName": "Blex, Vexing Pest", "finishes": ["nonfoil", "foil"], "flavorText": "Pests are typically harvested before they outgrow their enclosures. Blex is far from typical.", "foreignData": [], "frameEffects": ["legendary"], "frameVersion": "2015", "hasFoil": true, "hasNonFoil": true, "identifiers": {"cardKingdomFoilId": "244824", "cardKingdomId": "244519", "cardsphereId": "76146", "mcmId": "558102", "mcmMetaId": "339152", "mtgArenaId": "76542", "mtgjsonV4Id": "f28ad430-f3bd-5e58-8a6d-ca8d6eedf381", "mtgoId": "88775", "multiverseId": "513627", "scryfallId": "c204b7ca-0904-40fa-b20c-92400fae20b8", "scryfallIllustrationId": "59e38d12-40fd-46ec-94ca-6ac4b09df2fc", "scryfallOracleId": "b2c53124-e422-4786-8135-67c350d71a36", "tcgplayerProductId": "235972"}, "language": "English", "layout": "modal_dfc", "leadershipSkills": {"brawl": false, "commander": true, "oathbreaker": false}, "legalities": {"commander": "Legal", "duel": "Legal", "explorer": "Legal", "gladiator": "Legal", "historic": "Legal", "historicbrawl": "Legal", "legacy": "Legal", "modern": "Legal", "oathbreaker": "Legal", "pioneer": "Legal", "vintage": "Legal"}, "manaCost": "{2}{G}", "manaValue": 3.0, "name": "Blex, Vexing Pest // Search for Blex", "number": "148", "otherFaceIds": ["55cd326c-01c3-5523-b2fe-22a614594d5c"], "power": "3", "printings": ["PRM", "PSTX", "STX"], "purchaseUrls": {"cardKingdom": "https://mtgjson.com/links/166ba31249f0ef45", "cardKingdomFoil": "https://mtgjson.com/links/bf3342d9490a7b11", "cardmarket": "https://mtgjson.com/links/987e61793c3e42bb", "tcgplayer": "https://mtgjson.com/links/7b2c31f2567edb6a"}, "rarity": "mythic", "rulings": [{"date": "2021-04-16", "text": "If a creature is more than one of the creature types Blex cares about, it may haunt your dreams, but it will get the +1/+1 bonus only once."}, {"date": "2021-04-16", "text": "While resolving Search for Blex, you may put any number of the cards into your hand, even if you don’t have enough life to cover it. We’ll assume you have a plan that’s better than “and then I’ll lose the game.”"}], "securityStamp": "oval", "setCode": "STX", "side": "a", "sourceProducts": {"foil": ["141e45d1-be58-5312-82ad-3cd774a567cb", "52fca41e-cac5-5eb2-b107-0164e829d064", "64143552-b778-5cdb-a29f-af955ef626e8", "7a22deb5-1740-5a47-af47-dcdc175c4b77", "b18d1de8-64f5-5aa6-bfd1-638520cc9c28"], "nonfoil": ["141e45d1-be58-5312-82ad-3cd774a567cb", "608dbc7d-9ae2-5c72-a0a1-c52e954bacc0", "64143552-b778-5cdb-a29f-af955ef626e8", "7a22deb5-1740-5a47-af47-dcdc175c4b77", "b18d1de8-64f5-5aa6-bfd1-638520cc9c28"]}, "subtypes": ["Pest"], "supertypes": ["Legendary"], "text": "Other Pests, Bats, Insects, Snakes, and Spiders you control get +1/+1.\nWhen Blex, Vexing Pest dies, you gain 4 life.", "toughness": "2", "type": "Legendary Creature — Pest", "types": ["Creature"], "uuid": "5a52cce7-3897-5901-a0ab-d1ed087c41ee", "variations": ["70c713a0-f8a4-5980-9f69-4580764c3955"], "watermark": "witherbloom"} /CARD
Found! "name": "Blex, Vexing Pest // Search for Blex",
Card 10 {"artist": "Ekaterina Burmak", "artistIds": ["2a9f3780-fcc8-4ec5-b403-ee1d1370d3b4"], "availability": ["arena", "mtgo", "paper"], "boosterTypes": ["default"], "borderColor": "black", "colorIdentity": ["B", "G"], "colors": ["B"], "convertedManaCost": 4.0, "edhrecRank": 5557, "faceName": "Search for Blex", "finishes": ["nonfoil", "foil"], "foreignData": [], "frameEffects": ["legendary"], "frameVersion": "2015", "hasFoil": true, "hasNonFoil": true, "identifiers": {"cardsphereId": "76146", "mcmId": "558102", "mcmMetaId": "339152", "mtgArenaId": "76542", "mtgjsonV4Id": "45ac493c-9bdf-52d1-bee6-daaa33cbd11e", "mtgoId": "88775", "multiverseId": "513626", "scryfallId": "c204b7ca-0904-40fa-b20c-92400fae20b8", "scryfallIllustrationId": "a9acd45f-85da-4dad-8e85-cd0be900ce3c", "scryfallOracleId": "b2c53124-e422-4786-8135-67c350d71a36", "tcgplayerProductId": "235972"}, "language": "English", "layout": "modal_dfc", "legalities": {"commander": "Legal", "duel": "Legal", "explorer": "Legal", "gladiator": "Legal", "historic": "Legal", "historicbrawl": "Legal", "legacy": "Legal", "modern": "Legal", "oathbreaker": "Legal", "pioneer": "Legal", "vintage": "Legal"}, "manaCost": "{2}{B}{B}", "manaValue": 4.0, "name": "Blex, Vexing Pest // Search for Blex", "number": "148", "otherFaceIds": ["5a52cce7-3897-5901-a0ab-d1ed087c41ee"], "printings": ["PRM", "PSTX", "STX"], "purchaseUrls": {"cardmarket": "https://mtgjson.com/links/a4ecd3e83ad64f0b", "tcgplayer": "https://mtgjson.com/links/c7dc948bba877f5f"}, "rarity": "mythic", "rulings": [{"date": "2021-04-16", "text": "If a creature is more than one of the creature types Blex cares about, it may haunt your dreams, but it will get the +1/+1 bonus only once."}, {"date": "2021-04-16", "text": "While resolving Search for Blex, you may put any number of the cards into your hand, even if you don’t have enough life to cover it. We’ll assume you have a plan that’s better than “and then I’ll lose the game.”"}], "securityStamp": "oval", "setCode": "STX", "side": "b", "subtypes": [], "supertypes": [], "text": "Look at the top five cards of your library. You may put any number of them into your hand and the rest into your graveyard. You lose 3 life for each card you put into your hand this way.", "type": "Sorcery", "types": ["Sorcery"], "uuid": "55cd326c-01c3-5523-b2fe-22a614594d5c", "variations": ["caa57405-19ac-5d47-923a-af2a132cc509"], "watermark": "witherbloom"} /CARD
Found! "name": "Blex, Vexing Pest // Search for Blex",
Card 11 {"artist": "Ekaterina Burmak", "artistIds": ["2a9f3780-fcc8-4ec5-b403-ee1d1370d3b4"], "availability": ["arena", "mtgo", "paper"], "borderColor": "black", "colorIdentity": ["B", "G"], "colors": ["G"], "convertedManaCost": 3.0, "edhrecRank": 5557, "faceName": "Blex, Vexing Pest", "finishes": ["nonfoil", "foil"], "foreignData": [], "frameEffects": ["legendary", "extendedart"], "frameVersion": "2015", "hasFoil": true, "hasNonFoil": true, "identifiers": {"cardKingdomFoilId": "244943", "cardKingdomId": "244532", "cardsphereId": "76310", "mcmId": "558272", "mcmMetaId": "339152", "mtgjsonV4Id": "f074ffea-7ef9-5467-a232-ca4bf8ac8a6d", "multiverseId": "516911", "scryfallId": "930d3b70-7e8e-4a39-b60d-7d14196912e0", "scryfallIllustrationId": "59e38d12-40fd-46ec-94ca-6ac4b09df2fc", "scryfallOracleId": "b2c53124-e422-4786-8135-67c350d71a36", "tcgplayerProductId": "235973"}, "isStarter": true, "language": "English", "layout": "modal_dfc", "leadershipSkills": {"brawl": false, "commander": true, "oathbreaker": false}, "legalities": {"commander": "Legal", "duel": "Legal", "explorer": "Legal", "gladiator": "Legal", "historic": "Legal", "historicbrawl": "Legal", "legacy": "Legal", "modern": "Legal", "oathbreaker": "Legal", "pioneer": "Legal", "vintage": "Legal"}, "manaCost": "{2}{G}", "manaValue": 3.0, "name": "Blex, Vexing Pest // Search for Blex", "number": "322", "otherFaceIds": ["caa57405-19ac-5d47-923a-af2a132cc509"], "power": "3", "printings": ["PRM", "PSTX", "STX"], "promoTypes": ["boosterfun"], "purchaseUrls": {"cardKingdom": "https://mtgjson.com/links/c1505b1cc91f2cb8", "cardKingdomFoil": "https://mtgjson.com/links/8f734f0ef72083d9", "cardmarket": "https://mtgjson.com/links/c85351a6e94aeeee", "tcgplayer": "https://mtgjson.com/links/8924a6dfe23f380e"}, "rarity": "mythic", "rulings": [{"date": "2021-04-16", "text": "If a creature is more than one of the creature types Blex cares about, it may haunt your dreams, but it will get the +1/+1 bonus only once."}, {"date": "2021-04-16", "text": "While resolving Search for Blex, you may put any number of the cards into your hand, even if you don’t have enough life to cover it. We’ll assume you have a plan that’s better than “and then I’ll lose the game.”"}], "securityStamp": "oval", "setCode": "STX", "side": "a", "sourceProducts": {"foil": ["52fca41e-cac5-5eb2-b107-0164e829d064"], "nonfoil": ["52fca41e-cac5-5eb2-b107-0164e829d064"]}, "subtypes": ["Pest"], "supertypes": ["Legendary"], "text": "Other Pests, Bats, Insects, Snakes, and Spiders you control get +1/+1.\nWhen Blex, Vexing Pest dies, you gain 4 life.", "toughness": "2", "type": "Legendary Creature — Pest", "types": ["Creature"], "uuid": "70c713a0-f8a4-5980-9f69-4580764c3955", "variations": ["5a52cce7-3897-5901-a0ab-d1ed087c41ee"], "watermark": "witherbloom"} /CARD
Found! "name": "Blex, Vexing Pest // Search for Blex",
Card 12 {"artist": "Ekaterina Burmak", "artistIds": ["2a9f3780-fcc8-4ec5-b403-ee1d1370d3b4"], "availability": ["arena", "mtgo", "paper"], "borderColor": "black", "colorIdentity": ["B", "G"], "colors": ["B"], "convertedManaCost": 4.0, "edhrecRank": 5557, "faceName": "Search for Blex", "finishes": ["nonfoil", "foil"], "foreignData": [], "frameEffects": ["legendary", "extendedart"], "frameVersion": "2015", "hasFoil": true, "hasNonFoil": true, "identifiers": {"cardsphereId": "76310", "mcmId": "558272", "mcmMetaId": "339152", "mtgjsonV4Id": "4f1e841e-138e-5aa7-8fd2-82a5512696a2", "multiverseId": "516910", "scryfallId": "930d3b70-7e8e-4a39-b60d-7d14196912e0", "scryfallIllustrationId": "a9acd45f-85da-4dad-8e85-cd0be900ce3c", "scryfallOracleId": "b2c53124-e422-4786-8135-67c350d71a36", "tcgplayerProductId": "235973"}, "isStarter": true, "language": "English", "layout": "modal_dfc", "legalities": {"commander": "Legal", "duel": "Legal", "explorer": "Legal", "gladiator": "Legal", "historic": "Legal", "historicbrawl": "Legal", "legacy": "Legal", "modern": "Legal", "oathbreaker": "Legal", "pioneer": "Legal", "vintage": "Legal"}, "manaCost": "{2}{B}{B}", "manaValue": 4.0, "name": "Blex, Vexing Pest // Search for Blex", "number": "322", "otherFaceIds": ["70c713a0-f8a4-5980-9f69-4580764c3955"], "printings": ["PRM", "PSTX", "STX"], "promoTypes": ["boosterfun"], "purchaseUrls": {"cardmarket": "https://mtgjson.com/links/9cdbceab8fd1b3dd", "tcgplayer": "https://mtgjson.com/links/e6203e9a428c38d6"}, "rarity": "mythic", "rulings": [{"date": "2021-04-16", "text": "If a creature is more than one of the creature types Blex cares about, it may haunt your dreams, but it will get the +1/+1 bonus only once."}, {"date": "2021-04-16", "text": "While resolving Search for Blex, you may put any number of the cards into your hand, even if you don’t have enough life to cover it. We’ll assume you have a plan that’s better than “and then I’ll lose the game.”"}], "securityStamp": "oval", "setCode": "STX", "side": "b", "subtypes": [], "supertypes": [], "text": "Look at the top five cards of your library. You may put any number of them into your hand and the rest into your graveyard. You lose 3 life for each card you put into your hand this way.", "type": "Sorcery", "types": ["Sorcery"], "uuid": "caa57405-19ac-5d47-923a-af2a132cc509", "variations": ["55cd326c-01c3-5523-b2fe-22a614594d5c"], "watermark": "witherbloom"} /CARD
```

And even single-faced regular cards lack the `faceManaValue` y'all promote over `manaValue` in the linked issue's PR:

```json
{"artist": "Alex Dos Diaz", "artistIds": ["d11a1b18-57ff-41d3-a3a5-9ac3914b42d1"], "availability": ["mtgo", "paper"], "borderColor": "borderless", "colorIdentity": ["R", "W"], "colors": ["R", "W"], "convertedManaCost": 7.0, "edhrecRank": 1110, "edhrecSaltiness": 0.85, "finishes": ["nonfoil", "foil"], "foreignData": [{"language": "German", "multiverseId": 629348, "name": "Sela Goldnachtschwert", "text": "Fliegend, Erstschlag\nFalls eine Quelle einem Gegner oder einer bleibenden Karte, die ein Gegner kontrolliert, Schaden zufügen würde, fügt sie jenem Spieler oder jener bleibenden Karte stattdessen doppelt so viele Schadenspunkte zu.\nFalls eine Quelle dir oder einer bleibenden Karte, die du kontrollierst, Schaden zufügen würde, verhindere die Hälfte jener Schadenspunkte, aufgerundet.", "type": "Legendäre Kreatur — Engel"}, {"language": "Spanish", "multiverseId": 629372, "name": "Gisela, Espada de la Noche Dorada", "text": "Vuela, daña primero.\nSi una fuente fuera a hacer daño a un oponente o a un permanente que controla un oponente, en vez de eso, esa fuente hace el doble de ese daño a ese jugador o permanente.\nSi una fuente fuera a hacerte daño a ti o a un permanente que controlas, prevén la mitad de ese daño, redondeando hacia arriba.", "type": "Criatura legendaria — Ángel"}, {"language": "French", "multiverseId": 629396, "name": "Gisela, lame d'Ornuit", "text": "Vol, initiative\nSi une source devait infliger des blessures à un adversaire ou à un permanent qu'un adversaire contrôle, cette source inflige le double de ces blessures à ce joueur ou à ce permanent à la place.\nSi une source devait infliger des blessures à vous ou à un permanent que vous contrôlez, prévenez la moitié de ces blessures, arrondie à l'unité supérieure.", "type": "Créature légendaire : ange"}, {"language": "Japanese", "multiverseId": 629444, "name": "黄金夜の刃、ギセラ", "text": "飛行、先制攻撃\n発生源１つが対戦相手や対戦相手がコントロールしているパーマネントにダメージを与えるなら、代わりにその発生源はそのプレイヤーやパーマネントにその点数の２倍のダメージを与える。\n発生源１つがあなたやあなたがコントロールしているパーマネントにダメージを与えるなら、そのダメージの端数を切り上げた半分を軽減する。", "type": "伝説のクリーチャー — 天使"}, {"language": "Portuguese (Brazil)", "multiverseId": 629468, "name": "Gisela, Espadachim dos Noite Dourada", "text": "Voar, iniciativa\nSe uma fonte for causar dano a um oponente ou a uma permanente que um oponente controla, em vez disso, ela causa o dobro do dano àquele jogador ou àquela permanente.\nSe uma fonte for causar dano a você ou a uma permanente que você controla, previna metade do dano, arredondada para cima.", "type": "Criatura Lendária — Anjo"}, {"language": "Chinese Simplified", "multiverseId": 629492, "name": "金夜之刃姬瑟拉", "text": "飞行，先攻\n如果某来源将向任一对手或由对手操控的永久物造成伤害，则改为该来源对该牌手或永久物造成两倍的伤害。\n如果某来源将对你或任一由你操控的永久物造成伤害，则防止该伤害的一半，小数点后进位。", "type": "传奇生物 ～天使"}], "frameEffects": ["legendary", "inverted"], "frameVersion": "2015", "hasFoil": true, "hasNonFoil": true, "identifiers": {"cardKingdomFoilId": "281139", "cardKingdomId": "279963", "cardsphereId": "106649", "mcmId": "723604", "mcmMetaId": "205316", "mtgjsonV4Id": "feab73f3-849b-594d-82ee-10de65224f21", "mtgoId": "113617", "multiverseId": "629324", "scryfallId": "477f06ee-b986-4de2-97e9-97ba7ec7c45d", "scryfallIllustrationId": "4ef2ca7f-5f26-4594-9240-caa8bee2207f", "scryfallOracleId": "66f9f325-5e8e-4ebf-b5b3-c6410d80f2c5", "tcgplayerProductId": "504706"}, "isReprint": true, "keywords": ["First strike", "Flying"], "language": "English", "layout": "normal", "leadershipSkills": {"brawl": false, "commander": true, "oathbreaker": false}, "legalities": {"commander": "Legal", "duel": "Legal", "legacy": "Legal", "modern": "Legal", "oathbreaker": "Legal", "vintage": "Legal"}, "manaCost": "{4}{R}{W}{W}", "manaValue": 7.0, "name": "Gisela, Blade of Goldnight", "number": "682", "originalText": "Flying, first strike\nIf a source would deal damage to an opponent or a permanent an opponent controls, that source deals double that damage to that player or permanent instead.\nIf a source would deal damage to you or a permanent you control, prevent half that damage, rounded up.", "originalType": "Legendary Creature — Angel", "power": "5", "printings": ["A25", "AVR", "C15", "CM2", "CMM", "PHEL", "PLIST", "SLD"], "promoTypes": ["boosterfun"], "purchaseUrls": {"cardKingdom": "https://mtgjson.com/links/0fab03d3d529a895", "cardKingdomFoil": "https://mtgjson.com/links/237ff4a5254eb309", "cardmarket": "https://mtgjson.com/links/475ec72e6b5aa3a9", "tcgplayer": "https://mtgjson.com/links/22cca8d0975639d0"}, "rarity": "rare", "rulings": [{"date": "2018-03-16", "text": "Gisela doubles damage dealt to opponents and permanents your opponents control from any source, including sources controlled by those opponents."}, {"date": "2018-03-16", "text": "If damage dealt by a source you control is being divided or assigned among multiple permanents an opponent controls or among an opponent and one or more permanents they control simultaneously, divide the original amount and double the results. For example, if you attack with a 5/5 creature with trample and your opponent blocks with a 2/2 creature, you can assign 2 damage to the blocker and 3 damage to the defending player. These amounts are then doubled to 4 and 6 damage, respectively. You can’t double the damage to 10 first and then assign 2 to the creature and 8 to the player."}, {"date": "2018-03-16", "text": "If multiple replacement effects would modify how damage would be dealt, the player being dealt damage (or the controller of the permanent being dealt damage) chooses the order in which to apply those effects."}], "securityStamp": "oval", "setCode": "CMM", "sourceProducts": {"foil": ["27cadb89-a295-5ce4-89e1-5854303185db", "b2bbfbd2-13ee-5cf9-9c3a-8cf33204a839", "b9f3d07f-8ad0-5def-8086-e5199e004d4d", "d03a0e5f-11c7-504a-a486-44179896f770"], "nonfoil": ["27cadb89-a295-5ce4-89e1-5854303185db", "b2bbfbd2-13ee-5cf9-9c3a-8cf33204a839", "b9f3d07f-8ad0-5def-8086-e5199e004d4d", "d03a0e5f-11c7-504a-a486-44179896f770"]}, "subtypes": ["Angel"], "supertypes": ["Legendary"], "text": "Flying, first strike\nIf a source would deal damage to an opponent or a permanent an opponent controls, that source deals double that damage to that player or permanent instead.\nIf a source would deal damage to you or a permanent you control, prevent half that damage, rounded up.", "toughness": "5", "type": "Legendary Creature — Angel", "types": ["Creature"], "uuid": "eee2a568-149d-5e9c-b596-17123eb5d492", "variations": ["89c6ba4d-0bd0-5b51-bf0c-42b63d74dfeb", "57cd427d-221a-5397-b8c6-e3eb7dfd908d"]}
```

As noted in the other issue, the free edition of PyCharm can process the JSON. VS Code and IntelliJ can't unless you run your own parser in them.

@ZeldaZach Please reopen this issue and/or approve the Cockatrice PR that uses `manaValue` instead of `faceManaValue`.

**Comment by tooomm** [comment]
>And even single-faced regular cards lack the faceManaValue

[`faceManaValue`](https://mtgjson.com/data-models/card/card-atomic/#facemanavalue) is an optional MTGJSON property, it does not makes sense to have it for normal, single sided cards. `manaValue` has all info in that case already.
That's only an assumption as the documentation does not tell us when it's present or not.

<br>

>There is still only one instance of manaValue (no faceManaValue) in the aforementioned cases, eg 12 of Blex:

The data you provided is difficult to read, I'm sorry.
But only one instance of `manaValue` sounds correct as there should always only be one value per "physical card", no?
As I understand it, all cards with several faces/sides/modes should have a `faceManaValue`.
So should "Search for Blex" and its back side "Blex, Vexing Pest" - but they apparently don't if I understand you correctly?

But I'm no MtG judge, and it's quite confusing.

**Comment by CTimmerman** [comment]
> [faceManaValue](https://mtgjson.com/data-models/card/card-atomic/#facemanavalue) is an optional MTGJSON property, it does not makes sense to have it for normal, single sided cards. manaValue has all info in that case already.
That's only an assumption as the documentation does not tell us when it's present or not.

According to the docs, `faceManaValue` is an optional field, so your PR update to only use it for the back face is correct, except maybe when the back face has a different MV as right now Cockatrice doesn't appear to check for the "b" side so would apply it to the "a" side as well?

> The data you provided is difficult to read, I'm sorry. But only one instance of `manaValue` sounds correct as there should always only be one value per "physical card", no? As I understand it, all cards with several faces/sides/modes should have a `faceManaValue`. So should "Search for Blex" and its back side "Blex, Vexing Pest" - but they apparently don't if I understand you correctly?

`manaValue` is indeed a required field, so i was surprised to find it missing in the first search result of my custom parser.
As Blex has two card faces: front face and back face; it should have a `backFaceManaValue`, but does not according to VS Code's JSON-language-features-formatted 1000+ line search results i compactly pasted in my previous comment.

> But I'm no MtG judge, and it's quite confusing.

I'm no MtG judge either, but https://mtg.fandom.com/wiki/Mana_value#:~:text=Modal has a nice overview of mana values and comprehensive rules.

Looks like Commit and Memory of "Commit // Memory" are two separate names but only Memory is stored separately:

```json
{
    "artist": "Ryan Alexander Lee",
    "artistIds": [
        "f4d194b4-958a-4830-8d87-1a84e0b9ffbc"
    ],
    "availability": [
        "arena"
    ],
    "boosterTypes": [
        "default"
    ],
    "borderColor": "black",
    "colorIdentity": [
        "U"
    ],
    "colors": [
        "U"
    ],
    "convertedManaCost": 10.0,
    "edhrecRank": 1906,
    "edhrecSaltiness": 0.26,
    "faceConvertedManaCost": 6.0,
    "faceManaValue": 6.0,
    "faceName": "Memory",
    "finishes": [
        "nonfoil"
    ],
    "foreignData": [],
    "frameVersion": "2015",
    "hasFoil": false,
    "hasNonFoil": true,
    "identifiers": {
        "mtgArenaId": "73828",
        "mtgjsonV4Id": "54fa229d-ad3d-5677-9cb9-67c800081963",
        "multiverseId": "547521",
        "scryfallId": "d8a99b6b-0ff6-4921-8ce4-eafe6c3493d0",
        "scryfallIllustrationId": "f99e2092-e035-4043-b813-9dd1d32de0a0",
        "scryfallOracleId": "56fd8895-3be2-4591-86fa-87567d9cdc14"
    },
    "isOnlineOnly": true,
    "isReprint": true,
    "keywords": [
        "Aftermath"
    ],
    "language": "English",
    "layout": "aftermath",
    "legalities": {
        "commander": "Legal",
        "duel": "Legal",
        "explorer": "Legal",
        "gladiator": "Legal",
        "historic": "Legal",
        "historicbrawl": "Legal",
        "legacy": "Legal",
        "modern": "Legal",
        "oathbreaker": "Legal",
        "penny": "Legal",
        "pioneer": "Legal",
        "vintage": "Legal"
    },
    "manaCost": "{4}{U}{U}",
    "manaValue": 10.0,
    "name": "Commit // Memory",
    "number": "54",
    "originalText": "Aftermath (Cast this spell only from your graveyard. Then exile it.)\nEach player shuffles his or her hand and graveyard into his or her library, then draws seven cards.",
    "originalType": "Sorcery",
    "otherFaceIds": [
        "f071b478-3865-539b-9837-fe6793820a84"
    ],
    "printings": [
        "AKH",
        "AKR",
        "LCC",
        "MB1",
        "NCC",
        "PAKH"
    ],
    "purchaseUrls": {},
    "rarity": "rare",
    "rulings": [
        {
            "date": "2017-04-18",
            "text": "A spell with aftermath cast from a graveyard will always be exiled afterward, whether it resolves, it's countered, or it leaves the stack in some other way."
        },
        {
            "date": "2017-04-18",
            "text": "All split cards have two card faces on a single card, and you put a split card onto the stack with only the half you're casting. The characteristics of the half of the card you didn't cast are ignored while the spell is on the stack. For example, if an effect prevents you from casting green spells, you can cast Destined of Destined // Lead, but not Lead."
        },
        {
            "date": "2017-04-18",
            "text": "Each split card has two names. If an effect instructs you to choose a card name, you may choose one, but not both."
        },
        {
            "date": "2017-04-18",
            "text": "Each split card is a single card. For example, if you discard one, you've discarded one card, not two. If an effect counts the number of instant and sorcery cards in your graveyard, Destined // Lead counts once, not twice."
        },
        {
            "date": "2017-04-18",
            "text": "If a copy of a spell or a token is put into its owner's library, it's moved there, then it will cease to exist as a state-based action."
        },
        {
            "date": "2017-04-18",
            "text": "If a spell is put into its owner's library, it's removed from the stack and thus will not resolve. The spell isn't countered; it just no longer exists. This works against a spell that can't be countered."
        },
        {
            "date": "2017-04-18",
            "text": "If another effect allows you to cast a split card with aftermath from a graveyard, you may cast either half. If you cast the half that has aftermath, you'll exile the card if it would leave the stack."
        },
        {
            "date": "2017-04-18",
            "text": "If another effect allows you to cast a split card with aftermath from any zone other than a graveyard, you can't cast the half with aftermath."
        },
        {
            "date": "2017-04-18",
            "text": "If the target's owner's library contains no cards, the target spell or permanent will be put into that player's library as the only card."
        },
        {
            "date": "2017-04-18",
            "text": "If you cast the first half of a split card with aftermath during your turn, you'll have priority immediately after it resolves. You can cast the half with aftermath from your graveyard before any player can take any other action if it's legal for you to do so."
        },
        {
            "date": "2017-04-18",
            "text": "Split cards with aftermath have a new frame treatment—the half you can cast from your hand is oriented the same as other cards you'd cast from your hand, while the half you can cast from your graveyard is a traditional split card half. This frame treatment is for your convenience and has no rules significance."
        },
        {
            "date": "2017-04-18",
            "text": "While not on the stack, the characteristics of a split card are the combination of its two halves. For example, Destined // Lead is a green and black card, it is both an instant card and a sorcery card, and its mana value is 6. This means that if an effect allows you to cast a card with mana value 2 from your hand, you can't cast Destined. This is a change from the previous rules for split cards."
        },
        {
            "date": "2017-07-14",
            "text": "Once you've started to cast a spell with aftermath from your graveyard, the card is immediately moved to the stack. Opponents can't try to stop the ability by exiling the card with an effect such as that of Crook of Condemnation."
        }
    ],
    "setCode": "AKR",
    "side": "b",
    "subtypes": [],
    "supertypes": [],
    "text": "Aftermath (Cast this spell only from your graveyard. Then exile it.)\nEach player shuffles their hand and graveyard into their library, then draws seven cards.",
    "type": "Sorcery",
    "types": [
        "Sorcery"
    ],
    "uuid": "ae3bbfe2-61cd-5231-a7c7-703124bd018d"
} 
```

In that case "faceManaValue" is only relevant on the stack.

2023-12-16 update: I see now that "faceManaValue" is correct as each face has its own entry, referenced via `otherFaceIds`.

**Comment by ZeldaZach** [comment]
face cmc is missing from some entities, will address

**Comment by tooomm** [comment]
>According to the docs, faceManaValue is an optional field, so your PR update to only use it for the back face is correct, except maybe when the back face has a different MV as right now Cockatrice doesn't appear to check for the "b" side so would apply it to the "a" side as well?

We should test this after the recent change and with the new data!

But maybe better to keep the conversation here focused on missing and wrong data in MTGJSON, not specific issues in downstream tools.

>manaValue is indeed a required field, so i was surprised to find it missing in the first search result of my custom parser.

Is that still the case? That sounds wrong.

<br>

Thanks a lot @ZeldaZach for improving the data!

**Comment by CTimmerman** [comment]
> Is that still the case? That sounds wrong.

Blex now has the proper data and shows correctly in Cockatrice. Thanks! https://github.com/Cockatrice/Cockatrice/issues/4271#issuecomment-1858814254

Meld cards like _Bruna, the Fading Light_ have no `faceManaValue` and MV in Cockatrice (64-bit)
Version 2.9.0 (2023-09-14), but `manaValue` is correct on both faces.

<details>

```
3d12f898-11fb-52c3-bbed-e76c8a9f199d;Bruna, the Fading Light // Brisela, Voice of Nightmares
{'artist': 'Clint Cearley',
 'artistIds': ['03b75350-d5a0-4e3a-b3b4-8d4a5554385b'],
 'availability': ['mtgo', 'paper'],
 'borderColor': 'black',
 'colorIdentity': ['W'],
 'colors': ['W'],
 'convertedManaCost': 7.0,
 'edhrecRank': 2217,
 'edhrecSaltiness': 0.22,
 'faceName': 'Bruna, the Fading Light',
 'finishes': ['foil'],
 'flavorText': "She now sees only Emrakul's visions.",
 'foreignData': [],
 'frameEffects': ['mooneldrazidfc'],
 'frameVersion': '2015',
 'hasFoil': True,
 'hasNonFoil': False,
 'identifiers': {'cardsphereId': '52596',
                 'mcmId': '312684',
                 'mcmMetaId': '222994',
                 'mtgjsonV4Id': 'd7e29b43-cff5-5d48-a56b-303991800b62',
                 'mtgoId': '66409',
                 'multiverseId': '439322',
                 'scryfallId': '2b45fb19-450d-40bd-91c7-b5ace4a77f2a',
                 'scryfallIllustrationId': '4c8cee4a-a9a4-42eb-9cbf-fcc6c6344d00',
                 'scryfallOracleId': 'bce7e45d-b2c5-43aa-91a6-3a3c24f30850',
                 'tcgplayerProductId': '151636'},
 'isReprint': True,
 'isStarter': True,
 'keywords': ['Flying', 'Vigilance'],
 'language': 'English',
 'layout': 'meld',
 'leadershipSkills': {'brawl': False, 'commander': True, 'oathbreaker': False},
 'legalities': {'commander': 'Legal',
                'duel': 'Legal',
                'explorer': 'Legal',
                'gladiator': 'Legal',
                'historic': 'Legal',
                'historicbrawl': 'Legal',
                'legacy': 'Legal',
                'modern': 'Legal',
                'oathbreaker': 'Legal',
                'penny': 'Legal',
                'pioneer': 'Legal',
                'timeless': 'Legal',
                'vintage': 'Legal'},
 'manaCost': '{5}{W}{W}',
 'manaValue': 7.0,
 'name': 'Bruna, the Fading Light // Brisela, Voice of Nightmares',
 'number': '5a',
 'originalText': 'When you cast Bruna, the Fading Light, you may return target '
                 'Angel or Human creature card from your graveyard to the '
                 'battlefield.\n'
                 'Flying, vigilance\n'
                 '(Melds with Gisela, the Broken Blade.)',
 'originalType': 'Legendary Creature — Angel Horror',
 'otherFaceIds': ['b3dc9b70-fe64-5370-841b-5282d00fb1a9'],
 'power': '5',
 'printings': ['EMN', 'PEMN', 'SIR', 'SLD', 'V17'],
 'purchaseUrls': {'cardmarket': 'https://mtgjson.com/links/517eda6703caf95c',
                  'tcgplayer': 'https://mtgjson.com/links/1e061e31f4b71707'},
 'rarity': 'mythic',
 'rulings': [{'date': '2016-07-13',
              'text': 'A “when you cast” triggered ability resolves before the '
                      'original spell resolves. It resolves even if the '
                      'original spell is countered, and the original spell '
                      'resolves even if the triggered ability is countered.'},
             {'date': '2016-07-13',
              'text': 'Effects that increase or reduce the cost to cast a '
                      'spell (such as those of escalate and emerge) don’t '
                      'affect the spell’s mana value, so they won’t change '
                      'whether Brisela’s last ability restricts that spell '
                      'from being cast.'},
             {'date': '2016-07-13',
              'text': 'For more information on meld cards, see the Eldritch '
                      'Moon mechanics article '
                      '(https://magic.wizards.com/en/articles/archive/feature/eldritch-moon-mechanics-2016-06-27).'},
             {'date': '2016-07-13',
              'text': 'For spells with {X} in their mana costs, use the value '
                      'chosen for X to determine if the spell’s mana value is '
                      '3 or less. For example, your opponent could cast Burn '
                      'from Within (a spell with mana cost {X}{R}) with X '
                      'equal to 3, but not with X equal to 2.'},
             {'date': '2016-07-13',
              'text': 'In a Commander game, your commander may be Bruna, the '
                      'Fading Light or Gisela, the Broken Blade, and the other '
                      'may be in your deck. If they meld into Brisela, Voice '
                      'of Nightmares, Brisela will also be your commander; but '
                      'if Brisela leaves the battlefield, only the card chosen '
                      'as your commander at the start of the game may be put '
                      'into the command zone.'}],
 'securityStamp': 'oval',
 'setCode': 'V17',
 'side': 'a',
 'sourceProducts': {'foil': ['295c7d1a-c80b-5f09-a032-79d8e54f8453']},
 'subtypes': ['Angel', 'Horror'],
 'supertypes': ['Legendary'],
 'text': 'When you cast this spell, you may return target Angel or Human '
         'creature card from your graveyard to the battlefield.\n'
         'Flying, vigilance\n'
         '(Melds with Gisela, the Broken Blade.)',
 'toughness': '7',
 'type': 'Legendary Creature — Angel Horror',
 'types': ['Creature'],
 'uuid': '3d12f898-11fb-52c3-bbed-e76c8a9f199d'}
 
{'artist': 'Clint Cearley',
 'artistIds': ['03b75350-d5a0-4e3a-b3b4-8d4a5554385b'],
 'availability': ['mtgo', 'paper'],
 'borderColor': 'black',
 'colorIdentity': ['W'],
 'colors': [],
 'convertedManaCost': 11.0,
 'faceName': 'Brisela, Voice of Nightmares',
 'finishes': ['foil'],
 'flavorText': 'Upon discovering what had become of her sisters, Sigarda could '
               'only weep.',
 'foreignData': [],
 'frameEffects': ['mooneldrazidfc'],
 'frameVersion': '2015',
 'hasFoil': True,
 'hasNonFoil': False,
 'identifiers': {'mtgjsonV4Id': '458663a7-b298-5d84-a441-49c06608a536',
                 'multiverseId': '439323',
                 'scryfallId': 'eba8bb03-6093-4e2b-99a2-a3fc5d8eb659',
                 'scryfallIllustrationId': '9528e065-5297-4d91-bf75-3c175b54f0d4',
                 'scryfallOracleId': 'b23587e0-9fb7-492e-8bb4-f218e8e4ce6c'},
 'isReprint': True,
 'isStarter': True,
 'keywords': ['First strike', 'Flying', 'Lifelink', 'Vigilance'],
 'language': 'English',
 'layout': 'meld',
 'legalities': {'commander': 'Legal',
                'duel': 'Legal',
                'explorer': 'Legal',
                'gladiator': 'Legal',
                'historic': 'Legal',
                'historicbrawl': 'Legal',
                'legacy': 'Legal',
                'modern': 'Legal',
                'oathbreaker': 'Legal',
                'pioneer': 'Legal',
                'timeless': 'Legal',
                'vintage': 'Legal'},
 'manaValue': 11.0,
 'name': 'Brisela, Voice of Nightmares',
 'number': '5b',
 'originalText': 'When you cast Bruna, the Fading Light, you may return target '
                 'Angel or Human creature card from your graveyard to the '
                 'battlefield.\n'
                 'Flying, vigilance\n'
                 '(Melds with Gisela, the Broken Blade.)',
 'originalType': 'Legendary Creature — Angel Horror',
 'otherFaceIds': ['3d12f898-11fb-52c3-bbed-e76c8a9f199d',
                  'f0c70297-7170-5d45-a279-3685f9f0e3ad'],
 'power': '9',
 'printings': ['EMN', 'PEMN', 'SIR', 'SLD', 'V17'],
 'purchaseUrls': {},
 'rarity': 'mythic',
 'rulings': [{'date': '2016-07-13',
              'text': 'A “when you cast” triggered ability resolves before the '
                      'original spell resolves. It resolves even if the '
                      'original spell is countered, and the original spell '
                      'resolves even if the triggered ability is countered.'},
             {'date': '2016-07-13',
              'text': 'Effects that increase or reduce the cost to cast a '
                      'spell (such as those of escalate and emerge) don’t '
                      'affect the spell’s mana value, so they won’t change '
                      'whether Brisela’s last ability restricts that spell '
                      'from being cast.'},
             {'date': '2016-07-13',
              'text': 'For more information on meld cards, see the Eldritch '
                      'Moon mechanics article '
                      '(https://magic.wizards.com/en/articles/archive/feature/eldritch-moon-mechanics-2016-06-27).'},
             {'date': '2016-07-13',
              'text': 'For spells with {X} in their mana costs, use the value '
                      'chosen for X to determine if the spell’s mana value is '
                      '3 or less. For example, your opponent could cast Burn '
                      'from Within (a spell with mana cost {X}{R}) with X '
                      'equal to 3, but not with X equal to 2.'},
             {'date': '2016-07-13',
              'text': 'In a Commander game, your commander may be Bruna, the '
                      'Fading Light or Gisela, the Broken Blade, and the other '
                      'may be in your deck. If they meld into Brisela, Voice '
                      'of Nightmares, Brisela will also be your commander; but '
                      'if Brisela leaves the battlefield, only the card chosen '
                      'as your commander at the start of the game may be put '
                      'into the command zone.'}],
 'setCode': 'V17',
 'side': 'b',
 'subtypes': ['Eldrazi', 'Angel'],
 'supertypes': ['Legendary'],
 'text': 'Flying, first strike, vigilance, lifelink\n'
         "Your opponents can't cast spells with mana value 3 or less.",
 'toughness': '10',
 'type': 'Legendary Creature — Eldrazi Angel',
 'types': ['Creature'],
 'uuid': 'b3dc9b70-fe64-5370-841b-5282d00fb1a9'}
 ```
</details>

**Comment by tooomm** [comment]
What's the MTGJSON version you checked? I assume `v5.2.2+20231215`? @CTimmerman 

I did create a new issue to discuss meld cards and what data they should have: https://github.com/mtgjson/mtgjson/issues/1165

**Comment by CTimmerman** [comment]
Yes, 5.2.2+20231215. I assume "face" is an entire surface of a card, whereas "side" can be half a face as in split cards. If that is correct, then "faceManaValue" looks inconsistent to me. https://github.com/Cockatrice/Cockatrice/issues/4271#issuecomment-1861068872

**Comment by CTimmerman** [comment]
#NotAllDFC according to https://github.com/Cockatrice/Cockatrice/issues/4271#issuecomment-1917262449 - Reopen this or create a new issue or two?

**Comment by ZeldaZach** [comment]
This should be addressed in v5.3.0
